src/query/ts/m3db/encoded_step_iterator_test.go

```diff
@@ -509,7 +509,7 @@ func benchmarkNextIteration(b *testing.B, iterations int, t iterType) {
 
 		if v := profilesTaken[key]; v == 2 {
 			p := profile.Start(profile.MemProfile)
-			defer p.Stop()
+			p.Stop()
```

Comment:
Think this should still be `defer p.Stop()` since we want the memory profile to be taken at end of execution.

Reply:
Oops yeah I had been playing around with this as well but didn't mean to commit. Will revert.